packages/@react-spectrum/searchwithin/stories/SearchWithin.stories.tsx

```diff
@@ -0,0 +1,78 @@
+ * governing permissions and limitations under the License.
+ */
+import {action} from '@storybook/addon-actions';
+import {Item, Picker} from '@react-spectrum/picker';
+import React from 'react';
+import {SearchField} from '@react-spectrum/searchfield';
+import {SearchFieldProps} from '@react-types/searchfield';
+import {SearchWithin} from '../';
+import {SpectrumPickerProps} from '@react-types/select';
+import {SpectrumSearchWithinProps} from '@react-types/searchwithin';
+
+export default {
+  title: 'SearchWithin'
+};
+
+function render(props: Omit<SpectrumSearchWithinProps, 'children'> = {}, searchFieldProps: SearchFieldProps = {}, pickerProps: Omit<SpectrumPickerProps<object>, 'children'> = {}) {
+  return (
+    <SearchWithin label="Search" {...props}>
+      <SearchField placeholder="Search" {...searchFieldProps} onChange={action('change')} onSubmit={action('submit')} />
+      <Picker defaultSelectedKey="all" {...pickerProps} onSelectionChange={action('selectionChange')}>
+        <Item key="all">All</Item>
+        <Item key="campaigns">Campaigns</Item>
+        <Item key="audiences">Audiences</Item>
+        <Item key="tags">Tags</Item>
+      </Picker>
+    </SearchWithin>
+  );
+}
+
+function renderReverse(props: Omit<SpectrumSearchWithinProps, 'children'> = {}, searchFieldProps: SearchFieldProps = {}, pickerProps: Omit<SpectrumPickerProps<object>, 'children'> = {}) {
+  return (
+    <SearchWithin label="Search" {...props}>
+      <Picker defaultSelectedKey="all" {...pickerProps} onSelectionChange={action('selectionChange')}>
+        <Item key="all">All</Item>
+        <Item key="campaigns">Campaigns</Item>
+        <Item key="audiences">Audiences</Item>
+        <Item key="tags">Tags</Item>
+      </Picker>
+      <SearchField placeholder="Search" {...searchFieldProps} {...searchFieldProps} onChange={action('change')} onSubmit={action('submit')} />
+    </SearchWithin>
+  );
+}
+
+export const Default = () => render({});
+
+export const isDisabled = () => render({isDisabled: true});
+isDisabled.storyName = 'isDisabled: true';
+
+export const isRequired = () => render({isRequired: true});
+isRequired.storyName = 'isRequired: true';
+
+export const isRequiredNecessityIndicatorLabel = () => render({isRequired: true, necessityIndicator: 'label'});
+isRequiredNecessityIndicatorLabel.storyName = 'isRequired: true, necessityIndicator \'label\' ';
+
+export const isRequiredFalse_necessityIndicator = () => render({isRequired: false, necessityIndicator: 'label'});
+isRequiredFalse_necessityIndicator.storyName = 'isRequired: false, necessityIndicator \'label\' ';
+
+export const PickerDisabled = () => render({}, {}, {isDisabled: true});
+
+export const CustomWidth = () => render({width: 300});
+
+export const LabelPositionSide = () => render({labelPosition: 'side'});
+LabelPositionSide.storyName = 'labelPosition: side';
+
+export const NoLabel = () => render({label: undefined, 'aria-label': 'Aria Label'});
```

Comment:
Why is the width change between default, labelPositionSide, and noLabel? I only see width set on the customWidth story?

Reply:
Looking into this now. The width looks correct for labelPositionSide, and noLabel, and custom widths. But wrong for all the default ones. Everything looks fine in the docs so I think it's related to the storybook container.

Reply:
A width is being added at the field level when the top label is used, by a style from `.fieldlabel_spectrum-Field .fieldlabel_spectrum-Field--positionTop`. This width is 192px, and the problem is that our min width for the whole searchwithin should be 240. Looking into who we should go about this now.